one-click script for Azure

silentrvnning opened the pull request:
Script to instantiate a Taraxa node running on Azure cloud

rattrap commented:
Thanks for this @silentrvnning.

I tested it and it only seems to work on debian-based Linux distros.
Ca we make this universal and make sure it also works on MacOS?

silentrvnning commented:
Yep, modified the script and it should work on MacOS now. Added also support for the general install script, which should work for more distros. Added also some error checking for various commands.

rattrap commented on scripts/one-click-Azure.sh:
```diff
@@ -0,0 +1,173 @@
+fi
+
+# open ports
+az vm open-port --resource-group ${AZ_GROUP_NAME} --name ${AZ_APP_SERVICE_NAME} --port 7777,8777,10022 > /dev/null
```
Ports should be: `3000,7777,8777,10002`

rattrap reviewed (changes requested):
Looks good! Just change the ports I mentioned and I will merge it!
